### PR TITLE
Add isInteger32

### PR DESCRIPTION
**Pull request by skShekhar**
This PR adds a predicate to determine if an integer is 32 bits. #1282

### CODE REVIEW

**Comment by skShekhar** [comment]
@char0n Can you review this PR

**Comment by char0n** [comment]
Thanks for contributing. I'll get to core review ASAP.

File: src/isInteger32.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import { curryN } from 'ramda';
+
+import toInteger32 from './toInteger32';
+
+/**
+ * Checks whether the passed value is a 32 bit integer.
```
Comment (char0n) [inline src/isInteger32.js:6]:
```suggestion
 * Checks whether the passed value is a signed 32 bit integer.
```

File: src/isInteger32.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+ *
+ * @func isFinite
+ * @memberOf RA
```
Comment (char0n) [inline src/isInteger32.js:9]:
```suggestion
 * @aliases isInt32
 * @memberOf RA
```

File: src/isInteger32.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+ * @since {@link https://char0n.github.io/ramda-adjunct/2.32.0|v2.32.0}
+ * @category Type
+ * @sig * -> Boolean
+ * @param {*} val The value to test
+ * @return {boolean}
+ * @see {@link RA.isInteger32|isInteger32}
```
Comment (char0n) [inline src/isInteger32.js:15]:
```suggestion
 * @see {@link RA.toInteger32|toInteger32}
```

File: src/isInteger32.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+ * @example
+ *
+ * RA.isInteger32(0); //=> true
+ * RA.isInteger32((-2) ** 31); //=> true
+ *
+ * RA.isInteger32(Infinity); //=> false
+ * RA.isInteger32(NaN); //=> false
+ * RA.isInteger32(2 ** 31); //=> false
+ */
+const _isInteger32 = curryN(1, (val) => toInteger32(val) === val);
```
Comment (char0n) [inline src/isInteger32.js:25]:
```suggestion
const isInteger32 = curryN(1, (val) => toInteger32(val) === val);
```

File: src/isInteger32.js [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+export default _isInteger32;
```
Comment (char0n) [inline src/isInteger32.js:27]:
```suggestion
export default isInteger32;
```

File: test/isInteger32.js [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+    assert.isFalse(RA.isInteger32(NaN));
+    assert.isFalse(RA.isInteger32(Infinity));
+    assert.isFalse(RA.isInteger32(-Infinity));
+    assert.isFalse(RA.isInteger32(0.1));
+    assert.isFalse(RA.isInteger32(Math.E));
+    assert.isFalse(RA.isInteger32('10'));
+    assert.isFalse(RA.isInteger32(false));
+    assert.isFalse(RA.isInteger32([]));
+    assert.isFalse(RA.isInteger32({}));
+  });
```
Comment (char0n) [inline test/isInteger32.js:35]:
```suggestion
  });
  
  it('should support placeholder to specify "gaps"', function () {
    const isInteger32 = RA.isInteger32(R.__);

    assert.isTrue(isInteger32(0));
  });
```

**Review by char0n — CHANGES_REQUESTED** [review]
Great start! Please take a look at my code review comments. We're still missing two major things: typescript typings and tests for them. See for example this PR for more information about how to create ones: https://github.com/char0n/ramda-adjunct/commit/e9d321aef8bf1200d9b5e3bccd59f787eb8f31f5

**Comment by skShekhar** [comment]
@char0n I've added missing typescript typing and tests for `isInteger32`. The review comments have also been addressed

File: src/isInteger32.js (hunk 3 above)
Comment (char0n) [reply src/isInteger32.js]: Just for reference: you can directly commit my suggestion by clicking on "Commit suggestion" button. More info here: https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/incorporating-feedback-in-your-pull-request

**Review by char0n — APPROVED** [review]
Looks great! Your work is going to be part of next RA release.

**Pull request merged**